{{cookiecutter.project_slug}}/roadmap/ignite_roadmap.md

```diff
@@ -0,0 +1,64 @@
+# {{ cookiecutter.project_name }} Roadmap
+
+This document lists general directions that core team is interested to see developed in {{ cookiecutter.project_name }}
+
+## Principal goals
+
+<<DESCRIBE THE PRINCIPALS GOALS OF THE PROJECT?>> For example:
+
+- continue maintaining high-quality, well-tested and documented modules.
```

Comment:
all the items for this and other sections should be replaced by a placeholder
the topics are ok!